conf/schemas.go

```diff
@@ -116,6 +117,19 @@ func ReadSchemas(file string) (Schemas, error) {
 		}
 		schema.Priority = int64(p)<<32 - int64(i) // to sort records with same priority by position in file
 
+		reorderBufferStr := sec.ValueOf("reorderBuffer")
+		if len(reorderBufferStr) > 0 {
+			reorderWindow, err := strconv.ParseUint(reorderBufferStr, 10, 32)
```

Comment:
also i see docker/docker-dev-custom-cfg-kafka/storage-aggregation.conf does not have the comment that scripts/config/storage-schemas.conf has